app/src/main/java/de/westnordost/streetcomplete/osm/RoadWidth.kt

```diff
@@ -113,6 +113,7 @@ private fun estimateCyclewaysWidth(tags: Map<String, String>, isLane: Boolean):
     return leftWidth + rightWidth
 }
 
-private const val BROAD_LANE = 4f
-private const val LANE = 3f
+// when estimating, use lowest probable width to no over-estimate the road width
+private const val BROAD_LANE = 3.5f
+private const val LANE = 2.5f
```

Comment:
I've read the wikipedia article, I think this is fair.

```suggestion
// rather under-estimate then over-estimate the road width
private const val BROAD_LANE = 3.75f // standard interstate/motorway width
private const val LANE = 2.75f // widest trucks are about 2.6m
```